Automattic/wp-calypso
# Site Migration: Add back button on import or migrate step

Related to #88252

## Proposed Changes
* Add the back button on the import or migrate screen


## Testing Instructions
*  Open the site migration identify step (/setup/site-migration/site-migration-identify?flags=onboarding/new-migration-flow&siteSlug=[YOUR_SITE])
* Set any WordPress URL (JN works where) on the field "Enter the URL of the site:"
* Go to the next step
* Click on the back button
* Check if the identify step is showing the previous set URL
* Set another site the site URL on the field "Enter the URL of the site:"  to another WordPress site
* Continue to next step and check if the "from=" query param was updated with same value set on the  "Enter the URL of the site:"
